### PR TITLE
FLS-827: HSRA themes order added

### PR DESCRIPTION
**Pull request by hamzabinkhalid**
https://mhclgdigital.atlassian.net/browse/FLS-827

### Change description
- "View entire application" feature in assessment needs ordered themes list

- [ ] Unit tests and other appropriate tests added or updated
- [ ] README and other documentation has been updated / added (if needed)
- [X] Commit messages are meaningful and follow good commit message guidelines (e.g. "FS-XXXX: Add margin to nav items preventing overlapping of logo")

### How to test
- Verify "View entire application" feature is working as expected for HSRA new rounds VR and RP

### CODE REVIEW

File: assess/config/themes_mapping.py [hunk 1]
```diff
@@ -122,3 +122,38 @@ def ordered_themes(fund_round_short_name):
         ]
     if fund_round_short_name == "CTDFCR1":
         return ["project_name", "organisation_name"]
+    if fund_round_short_name == "HSRAVR":
+        return [
+            "what-would-you-like-to-name-your-application?",
```
Comment (ksp37-dluhc) [inline assess/config/themes_mapping.py:127]: Is it worth pulling out the common themes into a shared variable? Might make it easier to see at a glance what is the same between both rounds. 

Happy either way.

**Review by ksp37-dluhc — DISMISSED** [review]

File: assess/config/themes_mapping.py (hunk 1 above)
Comment (albertkol) [reply assess/config/themes_mapping.py:127]: What is the themes mapping for? I thought the order of the themes is decided by the assessment config.

Comment (hamzabinkhalid) [reply assess/config/themes_mapping.py:127]: Good shout @ksp37-dluhc , will add the changes 👍

Comment (hamzabinkhalid) [reply assess/config/themes_mapping.py:127]: @albertkol this one is used to specify the order of information shown in the "View entire application" feature in assessment.

https://github.com/communitiesuk/funding-service-pre-award-frontend/blob/e457b3fc15a5a117962cf3e1d12b23b70d3c73ad/assess/themes/deprecated_theme_mapper.py#L271

Screenshot 2024-12-24 at 12 34 28

**Review by ksp37-dluhc — APPROVED** [review]

**Pull request merged**